components/Hamburger.tsx

```diff
@@ -17,7 +17,7 @@ function Hamburger(props: any) {
     },
     {
       id: 2,
-      href: '/profile',
+      href: '/profilePage',
```

Comment:
Pebble: This will 404 I changed the name to profile.tsx it is no longer profilePage